kge/job/train.py

```diff
@@ -88,6 +88,11 @@ def __init__(
         self.num_examples = None
         self.type_str: Optional[str] = None
 
+
+        #: Hooks run after training
```

Comment:
We is it run? Perhaps: have a pre_run and post_run hook on `Job` level instead

Reply:
ok makes sense